### PR TITLE
Officer Upgrade Injector now 200 PE

### PR DESCRIPTION
**Pull request by Ben10083**
## About The Pull Request

Officer Upgrade Injector cost now 200 from 400.

## Why It's Good For The Game

Officers are in a odd state where they are both vital to the facility running smoothly, yet also immensely fragile without substantial investment. This PR aims to reduce the cost to invest into officer(s) and allow them to get out of 20 stat hell easier

## Changelog
:cl:
balance: Officer Upgrade Injector now 200 PE
/:cl:

### CODE REVIEW

**Comment by CrabbytheCrab** [comment]
finally I wont spend 10000 pe getting to ex after a records core

**Comment by EgorDinamit** [comment]
> vital to the facility running smoothly

???

File: code/datums/quirks/good.dm
Comment (LanceSmites328) [inline code/datums/quirks/good.dm:1]: What's this doin here

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/datums/quirks/good.dm
Comment (Ben10083) [reply code/datums/quirks/good.dm:1]: https://github.com/vlggms/lobotomy-corp13/pull/1791 Is that, unsure why its shown as changed, did it get changed again?

**Comment by Kitsunemitsu** [comment]
400 isn't that expensive in the grand scheme of things. Goes double because the EO is in charge of officer injectors.
Along with this, it is still purchased at 400, and both the EO and RO have been enjoying a short renaissance

**Pull request closed without merging**